packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx

```diff
@@ -90,14 +93,21 @@ export class PersonaCoinBase extends BaseComponent<IPersonaCoinProps, IPersonaSt
     });
 
     const shouldRenderInitials = Boolean(
-      !this.state.isImageLoaded && ((showInitialsUntilImageLoads && imageUrl) || !imageUrl || this.state.isImageError || hideImage)
+      (!this.state.isImageLoaded && ((showInitialsUntilImageLoads && imageUrl) || !imageUrl || this.state.isImageError || hideImage)) ||
```

Comment:
This logic is getting pretty complex it might be worth pulling some of them out in a couple `const`s?

Reply:
Agree with this. I pulled out the longer Boolean expressions into two separate consts